packages/pcd-types/src/pcd.ts

```diff
@@ -143,6 +127,32 @@ export interface PCDPackage<
   deserialize(seralized: string): Promise<PCD<C, P>>;
 }
 
+/**
+ * When displaying a {@link PCD} in Zupass, the PCDUI methods will be used to generate a
+ * card and, optionally, a header (used only when {@link PCDPackage.getDisplayOptions} does
```

Comment:
I wonder if `getDisplayOptions` should also live in the PCD UI package? It doesn't depend on react, but it is a UI functionality.

Reply:
It could be. But I can also imagine other non-UI-related reasons for wanting to be able to get a human-readable name for something, such as for logging, and it's nice that the base PCD package can do this.